### PR TITLE
Add APM agent information into operator docs - PLEASE MERGE Oct 3 PM

### PR DESCRIPTION
**Pull request by hmstepanek**
## Give us some context

* Add agent configuration documentation links and links to docker hub into the agent operator docs.

### CODE REVIEW

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx [hunk 1]
```diff
@@ -262,6 +284,26 @@ To uninstall and delete the Kubernetes agent operator, run this command:
   >
     Installing two APM agents in the same application can potentially lead to unexpected issues. Therefore, we strongly recommend removing any existing instrumentation before installing the operator.
   </Collapser>
+
+  <Collapser
+    id="modify-conf-apm"
```
Comment (hmstepanek) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx:289]:
```suggestion
    id="custom-apm"
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx [hunk 2]
```diff
@@ -262,6 +284,26 @@ To uninstall and delete the Kubernetes agent operator, run this command:
...
+
+    * [Java](https://docs.newrelic.com/docs/apm/agents/java-agent/custom-instrumentation/java-custom-instrumentation/)
+    * [Node](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-custom-instrumentation/)
+    * [Python](https://docs.newrelic.com/docs/apm/agents/python-agent/custom-instrumentation/python-custom-instrumentation/)
+    * [.NET](https://docs.newrelic.com/docs/apm/agents/net-agent/custom-instrumentation/introduction-net-custom-instrumentation/)
+    * [Ruby](https://docs.newrelic.com/docs/apm/agents/ruby-agent/api-guides/ruby-custom-instrumentation/)
+  </Collapser>
+
+  <Collapser
+    id="modify-conf-apm"
```
Comment (hmstepanek) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx:302]:
```suggestion
    id="read-only-file-system"
```

**Review by TimPansino — APPROVED** [review]
Likes look good to me

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx [hunk 3]
```diff
@@ -100,13 +100,19 @@ Each namespace requires a two-step process to enable automatic instrumentation o
 
    Make sure to replace `<YOUR_NEW_RELIC_INGEST_LICENSE_KEY>` with your valid New Relic license key.
 
-2. Apply the following CRD to each namespace you want to auto instrument. Run this command:
+2. Apply the following Custom Resource Definition (CRD) to each namespace you want to auto instrument. Run this command:
```
Comment (akristen) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx:103]: K8s seems to stylize this as `CustomResourceDefinition`. Do we want to follow this?

Comment (akristen) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx:103]: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/

**Review by akristen — APPROVED** [review]
merging!

**Pull request merged**